### PR TITLE
Add .spec files to package as rpms.

### PR DESCRIPTION
**Pull request by jgrund**
In order to publish block-device-listener and device-scanner-daemon on
copr, we need to add .spec files to the packages.

### CODE REVIEW

**Comment by brianjmurrell** [comment]
Is there an expectation that at some point there will be a test added to confirm that the RPM build works?

**Comment by jgrund** [comment]
That's a good idea. I'll add that before this lands.

File: packages/block-device-listener/block-device-listener.spec [hunk 1]
```diff
@@ -35,5 +42,5 @@ rm -rf %{buildroot}
 %{nodejs_sitelib}/@mfl/block-device-listener
 
 %changelog
-* Mon May 8th 2017 Joe Grund <grundjoseph@gmail.com> - 0.1.0
+* Mon May 8 2017 Joe Grund <grundjoseph@gmail.com> - 0.1.0
```
Comment (brianjmurrell) [inline packages/block-device-listener/block-device-listener.spec]: s/0.1.0/0.1.1/

**Comment by jgrund** [comment]
spec moving to MFLD.

**Pull request closed without merging**